drivers/SmartThings/philips-hue/profiles/motion-sensor.yml

```diff
@@ -0,0 +1,16 @@
+name: motion-sensor
+components:
+- id: main
+  capabilities:
+  - id: motionSensor
+    version: 1
+  - id: battery
```

Comment:
tom pointed this out on one of doug's pr, but we typically put `battery` right above `refresh`